install_config/router/customized_haproxy_router.adoc

```diff
@@ -115,21 +115,24 @@ The *define* action names the file that will contain the processed template.
 [cols="2*", options="header"]
 |===
 |Function | Meaning
-|`*endpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit) []Endpoint*` | Returns the list of valid endpoints.
-|`*env(variable, default string) string*` | Tries to get the named environment variable from the pod. Returns the second
-argument if the variable cannot be read or is empty.
+|`*processEndpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit, action string) []Endpoint*` |
+Returns the list of valid endpoints. When action is "shuffle", the order of endpoints is randomized.
+|`*env(variable, default ...string) string*` | Tries to get the named environment variable from the pod.
+If it is not defined or empty, it returns the optional second argument. Otherwise, it returns an empty string.
```

Comment:
What do you think of this? "If the value is empty or not defined and one or more optional arguments are provided, `env` returns the first non-empty argument. Otherwise, it returns an empty string."

Reply:
@Miciah In practice there are 0 or 1 optional values. The environment variable either has a default or it doesn't.